src/components/bulkConcepts/container/BulkConceptsPage.jsx

```diff
@@ -67,7 +67,7 @@ export class BulkConceptsPage extends Component {
     const { searchingOn } = this.state;
     const { currentPage } = this.props;
     if (currentPage !== prevProps.currentPage) {
-      if (searchingOn) {
+      if (!searchingOn) {
```

Comment:
Are we able to search? Is the logic true? because I see the part of code that is responsible for searching is being called when `searchingOn` is `false`

Reply:
Yes, we are able to search. The logic is right.